cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -59,7 +59,21 @@ public enum Feature implements FormatFeature
          *
          * @since 2.20
          */
-        HANDLE_UNDEFINED_AS_EMBEDDED_OBJECT(false)
+        HANDLE_UNDEFINED_AS_EMBEDDED_OBJECT(false),
```

Comment:
Just realized that I'll probably want change naming here slightly; instead of "handle", either "expose" or maybe "read". Will go ahead and make that change for both new Feature names.

Reply:
I think it could be better; I'm not good at naming these feature flags, though, so thank you! :-D

Reply:
np, that's what I am ultimately responsible for :)
(not that it was bad name per se, just trying to keep uniform terminology etc)